aws/amazon-chime-sdk-js
# Adding events "meetingReconnected" signalingDropped" "receivingAudioDropped"  to eventDidReceive

**Issue #:**
N/A

**Description of changes:**
Added events meetingReconnected, signalingDropped, and receivingAudioDropped to `eventDidReceive` where as currently these events are pushed to meeting history.


**Testing**
Added 3 new Tests:
1. publish event for Signalling Dropped when Web Socket Failed
2. publish event for Signalling Dropped when Web Socket Closed
3. publish event for Signalling Dropped when Web Socket error

Updated Test:
notifies the connection is poor

1. Have you successfully run `npm run build:release` locally? 
Yes
2. How did you test these changes? 
    a) Run Browser Demo App.
    b) Connect to a meeting
    c) Use Network Link Conditioner to mimic 100% loss.
    d) Open console section in developers tab  and filter by "Received an event"
    e) we should receive "receivingAudioDropped" and  "signalingDropped". event.
    f) Enable wifi and we should receive "meetingReconnected" event.
    g) Check gamma ingestion logs and we should receive all the three events.

3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
Yes, Browser Demo 

4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No

5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
